helpers/deploymentConfig.ts

```diff
@@ -228,20 +287,57 @@ export const assets: Assets = {
       price: "159990000000000000000",
     },
   ],
+  sepolia: [
+    {
+      token: "WBTC",
+      address: "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b",
+      oracle: "chainlink",
+      price: "25000000000000000000000",
```

Comment:
Are we using the `price` field of `WBTC`, `WETH` and `USDC`. AFAIU we are setting the Chainlink feed in Sepolia for these tokens